src/plugins/intel_cpu/tests/unit/snippets_transformations/x64/lowered/brgemm_blocking.cpp

```diff
@@ -322,16 +285,25 @@ TEST_F(BrgemmCPUBlockingTest, AMX) {
         auto data_b = linear_ir_ref->push_node<ov::opset10::Parameter>(precision, input_shape_b);
         auto copy_b = linear_ir_ref->push_node<BrgemmCopyB>(data_b.second, precision, BRGEMM_TYPE::REPACKING_ONLY);
         const auto copy_b_expr = *copy_b.first;
-        init_expr_descriptors(copy_b_expr, {{full_dim, full_dim}, {full_dim, full_dim}});
+        init_expr_descriptors(copy_b_expr, {{full_dim, full_dim},
+                                            {full_dim, full_dim}});
 
-        auto scratch = linear_ir_ref->push_node<snippets::op::Buffer>(ov::Shape{BrgemmCPU::SCRATCH_BYTE_SIZE});
-        scratch.first->get()->set_loop_ids({0});
+        auto scratch = linear_ir_ref->push_node<snippets::op::Buffer>(ov::Shape {BrgemmCPU::SCRATCH_BYTE_SIZE});
+        scratch.first->get()->set_loop_ids({2, 1, 0});
 
-        auto brgemm = linear_ir_ref->push_node<BrgemmCPU>(data_a.second, copy_b.second, scratch.second, BRGEMM_TYPE::WITH_AMX);
+        auto brgemm = linear_ir_ref->push_node<BrgemmCPU>(data_a.second, copy_b.second, scratch.second,
+                                                          BRGEMM_TYPE::WITH_AMX);
         const auto& brgemm_expr = *brgemm.first;
-        init_expr_descriptors(brgemm_expr, {{m_blk, full_dim}, {full_dim, full_dim}, get_default_subtensor(), {m_blk, full_dim}});
-        create_brgemm_with_copy_b_loop_infos(linear_ir_ref, brgemm_expr, copy_b_expr, m, m_blk);
-        brgemm_expr->set_loop_ids({0});
+        init_expr_descriptors(brgemm_expr, {{m_blk, k_blk}, {k_blk, n_blk}, get_default_subtensor(), {m_blk, n_blk}});
+        create_brgemm_loop_infos(linear_ir_ref, brgemm_expr, m, 0, k, k_blk, n, n_blk);
+
+        std::vector<LoopPort> entries {LoopPort(brgemm_expr->get_input_port(0), true, 1),
+                                       LoopPort(brgemm_expr->get_input_port(1), false, 1)};
+        std::vector<LoopPort> exits {LoopPort(brgemm_expr->get_output_port(0), true, 1)};
+        auto handlers = BrgemmBlockingBase::get_default_blocking_loop_handlers(m, m_blk);
+        linear_ir_ref->get_loop_manager()->
+            add_loop_info(std::make_shared<ov::snippets::lowered::UnifiedLoopInfo>(m, m_blk, entries, exits, handlers));
+        brgemm_expr->set_loop_ids({2, 1, 0});
```

Comment:
Can we apply these changes inside `create_brgemm_loop_infos`?

Reply:
We can't because they are needed only for the AMX case